lib/ui/account/password_reentry_page.dart

```diff
@@ -80,19 +80,16 @@ class _PasswordReentryPageState extends State<PasswordReentryPage> {
           } on KeyDerivationError catch (e, s) {
             _logger.severe("Password verification failed", e, s);
             await dialog.hide();
-            final dialogUserChoice = await showChoiceDialog(
-              context,
-              "Recreate password",
-              "The current device is not powerful enough to verify your "
+            final dialogChoice = await showNewChoiceDialog(
```

Comment:
In general, context can be always the first parameter, which is not optional. Adding it as required name parameter increases the verbosity and it might be really need to be named parameter because there will be most of the time just one input param of type context.